configure.ac

```diff
@@ -3012,6 +3012,7 @@ AC_CHECK_TYPE([bool])
 #need the defines for PRId64
 AC_CHECK_SIZEOF(int64_t)
 AC_CHECK_SIZEOF(long)
+AC_CHECK_SIZEOF(long long)
```

Comment:
You are not adding any SIZEOF_LONG_LONG macros to the code. Is this actually needed?

The other ones are for the libcompat definition:
``
#elif SIZEOF_INT64_T > SIZEOF_LONG
#define PRId64 "lld"
``